.changes/unreleased/Fixed-20221219-180311.yaml

```diff
@@ -0,0 +1,5 @@
+kind: Fixed
```

Comment:
( ignore the position of this comment )
Lets run, `go run main.go batch patch`+`go run main.go merge` and we can also do a patch release for this fix.